micro_manager/micro_manager.py

```diff
@@ -29,7 +29,11 @@
 from .adaptivity.local_adaptivity import LocalAdaptivityCalculator
 from .domain_decomposition import DomainDecomposer
 from .micro_simulation import create_simulation_class
-from .interpolation import Interpolation
+
+try:
+    from .interpolation import Interpolation
+except ImportError:
+    Interpolation = None
```

Comment:
What is the purpose of this?

Reply:
If `sklearn` is not installed it cannot be imported. It does not make sense to import `Interpolation` without `sklearn`. If there is no try-except block the execution will fail with an `ImportError` because Python attempts to import something that cannot be imported. There might be a better way to handle optional dependency imports but that is a solution I found.